src/evict/evict_lru.c

```diff
@@ -2197,7 +2197,7 @@ __wt_cache_eviction_worker(WT_SESSION_IMPL *session, bool busy, u_int pct_full)
 	if (timer)
 		__wt_epoch(session, &enter);
 
-	for (init_evict_count = cache->pages_evict;; ret = 0) {
+	for (init_evict_count = cache->pages_evict, yield_count = 0;; ret = 0) {
```

Comment:
I am wondering if I add a WT_ERR(..) statement before the for loop here, would the compiler warn that yield_count is uninitialized. Probably it will and we are fine, otherwise it will be difficult to make this code future!